packages/playwright/docs/index.mdx

```diff
@@ -158,10 +158,8 @@ export default defineConfig({
     trace: "retain-on-failure",
   },
 
-  // Add Argos reporter only when it runs on CI
-  reporter: process.env.CI
-    ? [["list"], ["@argos-ci/playwright/reporter"]]
-    : "list",
+  // Add Argos reporter
+  reporter: [["list"], ["@argos-ci/playwright/reporter"]],
```

Comment:
Move « reporter » above « use » to match the original playwright config structure.